### PR TITLE
upgrade to latest pxblue and security issues

### PR DESCRIPTION
**Pull request by jeffvg**
#### Changes proposed in this Pull Request:
- Mapbox > react-demo
Update PX Blue packages to latest and use react-themes
42283 vulnerabilities found
Severity: 42248 Low | 22 Moderate | 13 High
- after upgrades
7 vulnerabilities found
Severity: 4 Low | 1 Moderate | 2 High

- Mapbox > angular-demo
Update PX Blue packages to latest and use angular-themes
674 vulnerabilities found
Severity: 642 Low | 6 Moderate | 26 High
- after upgrades
11 vulnerabilities found
Severity: 5 Low | 2 Moderate | 4 High

### CODE REVIEW

File: angular-demo/package.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
...
     "@mapbox/mapbox-gl-geocoder": "^4.4.1",
-    "@pxblue/angular-components": "1.1.1",
-    "@pxblue/colors": "1.0.13",
+    "@pxblue/angular-components": "3.0.1",
+    "@pxblue/angular-themes": "5.1.2",
+    "@pxblue/colors": "3.0.0",
     "@pxblue/mapbox": "^1.0.1",
-    "@pxblue/ng-progress-icons": "1.1.4",
-    "@pxblue/themes": "2.1.0",
+    "@pxblue/ng-progress-icons": "1.2.1",
```
Comment (joebochill) [inline angular-demo/package.json:32]: Use version ranges with carets ^ instead of fixed versions.

File: react-demo/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "@material-ui/core": "4.2.0",
     "@material-ui/icons": "4.2.1",
     "@pxblue/mapbox": "^1.0.1",
-    "@pxblue/themes": "2.0.3",
+    "@pxblue/react-themes": "5.0.2",
```
Comment (joebochill) [inline react-demo/package.json:9]: use a caret

**Review by joebochill — APPROVED** [review]

**Review by joebochill — COMMENTED** [review]
Looks fine let just make sure that we're not locked down on the versions in package.json.

File: angular-demo/package.json (hunk 1 above)
Comment (jeffvg) [reply angular-demo/package.json:32]: update coming

**Pull request merged**